roles/ha/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Assign ressources to HA Group
+  ansible.builtin.uri:
+    url: "https://{{ proxmox_api_host }}:8006/api2/json/cluster/ha/resources"
+    method: POST
+    body_format: json
+    status_code: [200]
+    return_content: true
+    validate_certs: no
```

Comment:
From a security standpoint, I believe certificates should be valid by default. Maybe a variable would help here? (default to yes)

Reply:
Done, i am not sur if there is a way to implement defaults at the collection level, so i created a defaults file at the role level.